src/api/app/controllers/person/token_controller.rb

```diff
@@ -36,6 +36,30 @@ def delete
       render_ok
     end
 
+    # PUT /person/<login>/token/<id>
+    def update
+      authorize @user, :update?
+
+      permitted_params = params.slice(:enabled).permit(:enabled)
+
+      if permitted_params.blank?
+        render_error status: 400,
+                     errorcode: 'invalid_token_attributes',
+                     message: 'Parameters to modify token attributes are missing.'
+        return
+      end
+
+      unless %(true, false).include?(permitted_params[:enabled])
```

Comment:
This is a `Token` validation.

Reply:
> Why would that be a problem?

A request with an `enabled` value different to 'true' or 'false' cound be treated wrongly as truthy, without erroring.

Reply:
After introducing the XML schema validation, this code is no longer needed. I removed it.